packages/webapp/src/pages/election/components.tsx

```diff
@@ -47,14 +47,17 @@ export const MembersPage = (props: Props) => {
                 {members.error && "Fail to load members"}
             </Container>
             <MembersGrid members={members.data}>
-                {(member) => (
+                {(member, index) => (
                     <VotingMemberChip
                         key={`voting-chips-${member.account}`}
                         member={member}
                         isSelected={selectedMember === member.account}
                         onSelect={() => setSelected(member.account)}
-                        votesReceived={member.account === "pip.edev" ? 5 : 0}
-                        votingFor="Philip Pip"
+                        votesReceived={
+                            member.account === "edenmember12" ? 5 : 0
+                        }
+                        votingFor="Test Member 5"
+                        userIsVotingFor={member.account === "edenmember13"}
```

Comment:
I assume all this is hard-coded because we don't have this data for some reason?
Please at least add a TODO here so we don't lose track of it, but I'd prefer this test data came from somewhere else, like a fixture.

Reply:
Yeah, this is all just a dummy components display that's probably already out of date in other ways. Good call, though; I'll add a TODO on that.